tests/unit/challenges/test_aws_utils.py

```diff
@@ -0,0 +1,199 @@
+            description="Description for test challenge",
+            terms_and_conditions="Terms and conditions for test challenge",
+            submission_guidelines="Submission guidelines for test challenge",
+            creator=self.challenge_host_team,
+            published=False,
+            is_registration_open=True,
+            enable_forum=True,
+            queue="test_queue",
+            anonymous_leaderboard=False,
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+            approved_by_admin=False,
+        )
+        self.challenge.slug = "{}-{}".format(
+            self.challenge.title.replace(" ", "-").lower(), self.challenge.pk
+        )[:199]
+        self.challenge.save()
+
+        self.challenge_host = ChallengeHost.objects.create(
+            user=self.user,
+            team_name=self.challenge_host_team,
+            status=ChallengeHost.ACCEPTED,
+            permissions=ChallengeHost.ADMIN,
+        )
+        self.challenge2 = Challenge.objects.create(
+            title="Test Challenge 2",
+            creator=self.challenge_host_team,
+            queue="test_queue_2",
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+        )
+        with self.settings(MEDIA_ROOT="/tmp/evalai"):
+            self.challenge_phase = ChallengePhase.objects.create(
+                name="Challenge Phase",
+                description="Description for Challenge Phase",
+                start_date=timezone.now() - timedelta(days=2),
+                end_date=timezone.now() + timedelta(days=1),
+                challenge=self.challenge,
+            )
+            self.challenge_phase2 = ChallengePhase.objects.create(
+                name="Challenge Phase 2",
+                description="Description for Challenge Phase",
+                start_date=timezone.now() - timedelta(days=2),
+                end_date=timezone.now() + timedelta(days=1),
+                challenge=self.challenge2,
+            )
+
+        self.client.force_authenticate(user=self.user)
+        self.ecs_client = boto3.client("ecs", region_name=os.environ.get("AWS_DEFAULT_REGION", "us-east-1"), aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"), aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),)
+
+
+@mock_ecs
+class BaseAdminCallClass(BaseTestClass):
+    def setUp(self):
+        aws_utils.COMMON_SETTINGS_DICT["CLUSTER"] = "cluster"
+        super(BaseAdminCallClass, self).setUp()
+
+        self.challenge3 = Challenge.objects.create(
+            title="Test Challenge 3",
+            creator=self.challenge_host_team,
+            queue="test_queue_3",
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+        )
+        self.ecs_client.create_cluster(clusterName="cluster")
```

Comment:
@takitsuse could you update this to `aws_utils.COMMON_SETTINGS_DICT["CLUSTER"]` ?

Reply:
I think I already have that code in line 111?

Reply:
Okay, nevermind. Could you update the imports ?

Reply:
Give me 1 minute, I am updating them.

Reply:
@yashdusing Done :+1: